### PR TITLE
Sync organization members for reputation

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Awarding reputation is dependent on having the organization members properly synced

### CODE REVIEW

**Comment by iHiD** [comment]
I'll leave you to add the sidekiq stuff to this.

File: lib/tasks/sync_pull_requests_reputation.rake [hunk 1]
```diff
@@ -1,5 +1,6 @@
 desc 'Sync pull requests reputation'
 task sync_pull_requests_reputation: :environment do
   Github::PullRequest::SyncRepos.()
+  Github::OrganizationMember::SyncMembers.()
   User::ReputationToken::AwardForPullRequests.()
 end
```
Comment (iHiD) [inline lib/tasks/sync_pull_requests_reputation.rake]: I think this rake task should change to be `FetchAndSyncAllPullRequestsReputationJob.perform_later`.

Comment (iHiD) [reply lib/tasks/sync_pull_requests_reputation.rake]: Logic being:
1. DRY
2. We can't create reputation on bastion anyway.

Comment (iHiD) [reply lib/tasks/sync_pull_requests_reputation.rake]: Actually, if `User::ReputationToken::AwardForPullRequests` now creates jobs to create (which I think it does) then we could change it to `perform_now` instead, so we can run the bits we **can** run on bastion there.

Comment (ErikSchierboom) [reply lib/tasks/sync_pull_requests_reputation.rake]: `User::ReputationToken::AwardForPullRequests` does indeed create jobs:

```
pull_requests.find_each do |pr|
  AwardReputationForPullRequestJob.perform_later(pr.data)
  ..
```

File: lib/tasks/fetch_and_sync_all_pull_requests_reputation.rake [hunk 2]
```diff
@@ -0,0 +1,4 @@
+desc 'Fetch and sync all pull requests reputation'
+task fetch_and_sync_all_pull_requests_reputation_job: :environment do
+  FetchAndSyncAllPullRequestsReputationJob.perform_now
```
Comment (ErikSchierboom) [inline lib/tasks/fetch_and_sync_all_pull_requests_reputation.rake:3]: I've also removed the organization members sync rake task, as that is also done here via `FetchAndSyncAllPullRequestsReputationJob`

**Comment by iHiD** [comment]
Sweet. Thanks! Let's check sidekiq in 15mins to make it's worked ok.

**Pull request merged**